config.md

```diff
@@ -141,12 +143,12 @@ Since the configuration JSON that gets hashed references hashes of each layer in
 
     - **diff_ids** *array*, REQUIRED
 
-       An array of layer content hashes (`DiffIDs`), in order from bottom-most to top-most.
+       An array of layer content hashes (`DiffIDs`), in stack order from bottom-most to top-most (i.e. from `diff_ids[0]` to `diff_ids[len(diff_ids)]`).
```

Comment:
I really don't like introducing the word "stack" because it addles my poor brain